src/QMCWaveFunctions/tests/test_RotatedSPOs.cpp

```diff
@@ -21,6 +21,7 @@
 #include "QMCWaveFunctions/EinsplineSetBuilder.h"
 #include "QMCWaveFunctions/RotatedSPOs.h"
 #include "checkMatrix.hpp"
+#include "FakeSPO.h"
 
 #include <stdio.h>
```

Comment:
stdio.h->cstdio

Reply:
This kind of change is starting to feel overly picky.   Part of my motivation for #4559 was to make these kind of small but widespread updates to the code base separately.   Then hopefully new code would pick up the changes by copying the updated code, and reviews would need to be less concerned about these kind of changes.

Reply:
Sorry about. Fair point. It is a line not part of this PR.
I actually just wanted to take a note as I was reading through the code base and raise awareness of developers.